Civi/Token/TokenRow.php

```diff
@@ -225,6 +226,10 @@ public function fill($format = NULL) {
               if ($entity == 'activity' && $field == 'details') {
                 $htmlTokens[$entity][$field] = $value;
               }
+              elseif (\CRM_Utils_Array::value('data_type', \CRM_Utils_Array::value($field, $entityFields['values'])) == 'Memo') {
+                // Memo fields aka custom fields of type Note are html.
+                $htmlTokens[$entity][$field] = $value;
```

Comment:
@seamuslee001 Could you review from a security perspective?  Are we ok to insert html directly into tokens?

Reply:
@aydun this is more likely to be considered secure if you use the `CRM_Utils_String::purifyHTML() function`

Reply:
Thanks @colemanw - changed now

Reply:
@colemanw @seamuslee001 with this change is it mergeable?